src/engine.rs

```diff
@@ -165,11 +165,17 @@ where
             loop {
                 match LogBatch::from_bytes(&mut buf, file_num, offset) {
                     Ok(Some(mut log_batch)) => {
-                        let entries_size = log_batch.entries_size();
+                        let mut encoded_size = 0;
+                        for item in &log_batch.items {
```

Comment:
Does this can reduce CPU usage?

Reply:
It does same thing as the old `entries_size` function.